specs/networking/rpc-interface.md

```diff
@@ -154,7 +157,11 @@ Client MAY send `goodbye` messages upon disconnection. The reason field MAY be o
 
 - `1`: Client shut down.
 - `2`: Irrelevant network.
-- `3`: Fault/error.
+- `3`: Breach of protocol.
+- `4`: Useless peer.
+- `5`: Already Connected.
+- `6`: Self-connection.
+- `7`: Fault/error.
```

Comment:
These are the RLPx error codes, do we need those? In particular `5` implies you cannot have more than one connection with another peer.